rpc/QueryIterator.java

```diff
@@ -67,7 +74,12 @@ protected T computeNext() {
             case RES_NOT_SET:
                 throw new GraknClientException(MISSING_RESPONSE.message(className(TransactionProto.Transaction.Res.class)));
             default:
-                return transformResponse.apply(res);
+                currIterator = transformResponse.apply(res).iterator();
```

Comment:
I think we can simplify the `default` block to just:
```java
currIterator = transformResponse.apply(res).iterator();
return computeNext();
```
because the `if` check is already handled on line 54.